FAQ.md

```diff
@@ -72,6 +72,18 @@ The send operation uses the **`receiverTimeout`** value as the amount of time it
 Since a single batch is being transferred by a single send operation, if the batch contains a large number events you 
 have to adjust the `receiverTimeout` to give enough time to the send operation to complete its entire transfer.
 
+
+**Why am I seeing: `"java.util.concurrent.TimeoutException: Futures timed out after [5 minutes]"?`**
+
+The connector uses blocking calls on Futures in several places and uses a default 5 minutes timeout to ensure the progress
+is not blocked indefinitely. Therefore, generally speaking, the Future time-out exception could happen due to different reasons.
+
+However, a known situation that may result in seeing this error is when the connector keeps recreating the cached receivers 
+because they move between different executor nodes. Note that the receiver recreation is expected behavior when 
+the receiving task for a specific eventhubs partition is moving from one executor to another. 
+
+In this case, increasing the spark locality can help to reduce/avoid recreating receivers. The spark locality can be 
+increased by assigning a higher value than the default 3s to the "spark.locality.wait" property.
```

Comment:
Let's make a recommendation of using 15 seconds.

Reply:
Done